setup.py

```diff
@@ -39,7 +39,7 @@
         "uvicorn",
         "Jinja2"
     ],
-    entry_points = {
-        'console_scripts': ['gradio=gradio.dev:main'],
+    entry_points={
+        'console_scripts': ['gradio=gradio.reload:user', 'gradio-dev=gradio.reload:dev']
```

Comment:
We shouldn't create a separate entry point. Instead this should be a flag:

`$gradio --dev`

Reply:
Different endpoints feels like a nice decoupling for me, and would allow auto complete for `gradio-dev` as well, I prefer this way. Might ask @aliabid94's thought as well.

```
1. gradio-dev demo/folder/run.py
2. gradio --dev demo/folder/run.py
3. gradio demo/folder/run.py
```

Reply:
Btw united gradio and gradio-dev commands, from now on only `gradio demo/folder/run.py` exists